hazelcast/src/main/resources/cluster.sh

```diff
@@ -262,9 +262,10 @@ if [ "$OPERATION" = "change-cluster-version" ]; then
     STATUS=$(echo "${response}" | sed -e 's/^.*"status"[ ]*:[ ]*"//' -e 's/".*//');
 
     if [ "$STATUS" = "fail" ];then
-       NEWVERSION=$(echo "${response}" | sed -e 's/^.*"version"[ ]*:[ ]*"//' -e 's/".*//');
-       if [ "$NEWVERSION" != "null" ]; then
-           echo "Cluster is in ${NEWVERSION}, change failed."
+       MESSAGE=$(echo "${response}" | sed -e 's/^.*"message"[ ]*:[ ]*"//' -e 's/".*//');
+       # when there is no "message" element in response, above sed process returns "{"
+       if [ "$MESSAGE" != "{" ]; then
+           echo "Cluster version change failed: ${MESSAGE}"
        else
             echo "An error occured while changing cluster version!";
```

Comment:
Shouldn't we show the response here? It seems quite informative, if I understand the issue correctly.

Reply:
If there was a `"message"` element in the returned JSON (which [should be always the case](https://github.com/vbekiaris/hazelcast/blob/673f032883917e6905e496fb80fcbaba5e713487/hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpPostCommandProcessor.java#L518-L520)), then the message value will be displayed. This branch is handling non-standard response (which may or may not be informative ;-) ). Should we show the complete response in this case?

Reply:
I understood the issue that the regex is wrong, so the version cannot be parsed. Not that there is no message at all.